edx/XBlock
# [BD-13] depr: deprecated the user_id attribute

## Description

The user_id should be fetched from the 'user' service, not stored in the Runtime.



This change should not affect Learners, Course Authors, or anyone else using edx-platform.

## Supporting information
* [Content Core Platform Simplication project](https://openedx.atlassian.net/wiki/spaces/AC/pages/3091629503/Content+Core+Platform+Simplification)
* [BLENDED-113](https://openedx.atlassian.net/browse/BLENDED-113) (edX internal ticket)
* [FAL-2247](https://tasks.opencraft.com/browse/FAL-2247) (OpenCraft internal ticket)

## Testing instructions

See https://github.com/edx/edx-platform/pull/28440 for testing instructions and sandbox.

## Deadline

None

## Other information

* This PR must be merged and a new version of XBlock tagged before https://github.com/edx/edx-platform/pull/28440 can be merged.
